### PR TITLE
Prepare shapr 0.2.0 for CRAN

### PR DESCRIPTION
**Pull request by martinju**
Mainly following https://www.marinedatascience.co/blog/2020/01/09/checklist-for-r-package-re-submissions-on-cran/ + some additional stuff

- [x]  Styling with styler::style_pkg() and lintr:lint_package()
- [x]  Update NEWS.md and cran-comments.md
- [x]  spelling with devtools::spell_check()

Run tests on
- [x] GitHub Actions (windows-latest): R 4.0
- [x] GitHub Actions (ubuntu-16.04): R 4.0, 3.6, 3.5
- [x] GitHub Actions (macOS-latest): R-release, 4.0 
- [x] win-builder (x86_64-w64-mingw32): R 4.0, 3.6, R-devel
- [x] local Ubuntu 18.04: R 3.6
- [x] local Windows 10: R 4.0
- [x] R-hub (windows-x86_64-devel): R-devel
- [ ] R-hub (ubuntu-gcc-release): R-release
- [ ] R-hub (fedora-gcc-devel): R-devel
- [x] R-hub (macos-highsierra-release-cran): R-release
- [x] local Ubuntu 18.04: R 3.6 (without packages in Suggests): 
```devtools::check(vignettes = FALSE, env_vars=c(`_R_CHECK_DEPENDS_ONLY_` = "true"))```

NOTE: Bump version to 0.2.0 after merging, but before submission

### CODE REVIEW

**Comment by martinju** [comment]
Whao! For some reason tests are suddenly passing for macOS :-)

**Review by aredelmeier — APPROVED** [review]

File: NEWS.md [hunk 1]
```diff
@@ -1,4 +1,22 @@
 
+# shapr 0.2.0
+
+* Minor CRAN release
+* Added the new dependence modeling approach "ctree" which handles categorical features in addition
+  to numerical ones from the paper https://doi.org/10.1007/978-3-030-57321-8_7
```
[the remark this answers is not in the document]
Comment (aredelmeier) [reply NEWS.md:6]: or possibly "our paper"

**Comment by martinju** [comment]
Skipping the R-hub ubuntu and fedora tests as there is downtime on those servers and we essentially got it covered elsewhere

**Comment by martinju** [comment]
@aredelmeier Submitted! Crossing fingers

**Pull request merged**